ffi/src/jwt.rs

```diff
@@ -1,3 +1,9 @@
+// We use #[deprecated] for `JwsAlg::ED22519` to tell users that this algorithm is invalid
+// (`JwsAlg::EdDSA` should be used instead) but we should keep it inside enum to allow decoding of
+// invalid tokens generated by other libraries such as `golang-jws`. However, to avoid compilation
+// warnings, we need to allow deprecated items inside this module.
+#![allow(deprecated)]
+
```

Comment:
I would prefer you put the `#[allow(deprecated)]` just above the concerned code so we don’t accidentally suppress other deprecation warnings. 
I did something similar in Devolutions Gateway to avoid accidental use of dangerous token validation:
https://github.com/Devolutions/devolutions-gateway/blob/1b04c90634833d0caae80d4ce52739840997e633/devolutions-gateway/src/rdp_pcb.rs#L30

Reply:
Ah. Well. Yeah, I suppose you could do that :sweat_smile:

Reply:
Well, it worked, but for `ffi/wasm` I kept this attribute: https://github.com/Devolutions/picky-rs/pull/222#discussion_r1213472985